src/components/PersonLink.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Person } from '../types';
+import { Link, NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person;
+  people: Person[];
+  isSelected?: boolean | null;
+};
+
+export const PersonLink: React.FC<Props> = ({ person, people, isSelected }) => {
+  const renderPersonName = (name: string | null) => {
+    if (!name) {
+      return '-';
+    }
+
+    const foundPerson = people.find(persons => persons.name === name);
+
+    return foundPerson ? (
+      <NavLink
+        to={`/people/${foundPerson.slug}`}
+        className={classNames({ 'has-text-danger': foundPerson.sex === 'f' })}
+      >
+        {name}
+      </NavLink>
+    ) : (
+      name
+    );
+  };
+
+  return (
+    <tr data-cy="person" className={isSelected ? 'has-background-warning' : ''}>
```

Comment:
Use `classnames` - https://www.npmjs.com/package/classnames
Here and in other places